### PR TITLE
Menu pages

### PR DESCRIPTION
**Pull request by MlulekiN**

### CODE REVIEW

File: src/test/java/pages/MenuPage.java [hunk 1]
```diff
@@ -7,4 +7,7 @@ public static By topMenu(String topMenu) {
         return By.xpath("//div/descendant::li[contains(@class,'nav-item')]/child::a[text()='" + topMenu + "']");
     }
 
+    public static By ContactUsHeading = By.xpath("//h2[text()='Contact Us']");
```
Comment (benedictnkosi) [inline src/test/java/pages/MenuPage.java]: this is not on the menu page

File: src/test/java/pages/MenuPage.java [hunk 2]
```diff
@@ -7,4 +7,7 @@ public static By topMenu(String topMenu) {
...
+    public static By homePage = By.xpath("(//div[@class='banner-info']/descendant::h5[contains(text(),'Comfortable')])[2]");
```
Comment (benedictnkosi) [inline src/test/java/pages/MenuPage.java]: this is not on the menu page

**Review by benedictnkosi — CHANGES_REQUESTED** [review]

File: src/test/java/pages/MenuPage.java (hunk 1 above)
Comment (MlulekiN) [reply src/test/java/pages/MenuPage.java]: Done

File: src/test/java/pages/MenuPage.java (hunk 2 above)
Comment (MlulekiN) [reply src/test/java/pages/MenuPage.java]: Done

**Review by benedictnkosi — APPROVED** [review]

**Pull request merged**